src/Templates/ProjectTemplates/CSharp/.NETCore/ConsoleApplication/NuGet.json

```diff
@@ -0,0 +1,11 @@
+{
+  "dependencies": {
+    "Microsoft.NETCore.App": {
+      "version": "1.0.0"
+    }
+  },
+  "frameworks": {
+    "netcoreapp1.0": {
```

Comment:
This one should go away and come from the project. It will only be needed if we have a fx specific depenedncy